os_net_config/cli.py

```diff
@@ -359,6 +382,44 @@ def main(argv=sys.argv, main_logger=None):
         return ExitCode.SUCCESS
 
 
+def apply_remove_config(remove_config, root_dir, noop):
+    nmstate_remove_config = []
+    ifcfg_remove_config = []
+    rm_ifcfg_provider = load_provider("ifcfg", noop, root_dir)
+    rm_nmstate_provider = load_provider("nmstate", noop, root_dir)
+    success = True
+    for rem_json in remove_config:
+        rem_json.update({"type": "remove_net_device"})
```

Comment:
Can we get the same from config.yaml instead of manually adding. So the templates look like 
remove_config:
    - type: remove_net_device
       dev_type: interface
       dev_name: eth1
    - type: remove_net_device
       dev_type: vlan
       dev_name: vlan100